.github/workflows/generate_matrix_page.yml

```
@@ -0,0 +1,136 @@
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch to check out (default is main)'
+        default: 'main'
+        required: false
+        type: string
+      pr_number:
+        description: 'PR number to process (leave empty to process all history)'
+        required: false
+        type: string
+      process_mode:
```

actionlint (1.7.4)

"options" can not be set to "process_mode" input because its input type is not "choice"

(events)